designer/src/fields.tsx

```diff
@@ -30,6 +30,70 @@ const fields: {[key: string]: FieldType} = {
     validationSchema: [['yup.string']],
     initialValue: '',
   },
+
+  // DATE FIELDS
+  // ===========
+  DateTimePicker: {
+    'component-namespace': 'faims-custom',
+    'component-name': 'DateTimePicker',
+    'type-returned': 'faims-core::String',
+    'component-parameters': {
+      label: 'Date time picker',
+      fullWidth: true,
+      helperText: '',
+      advancedHelperText: '',
+      variant: 'outlined',
+      required: false,
+    },
+    validationSchema: [['yup.string']],
+    initialValue: '',
+  },
+  DatePicker: {
+    'component-namespace': 'faims-custom',
+    'component-name': 'DatePicker',
+    'type-returned': 'faims-core::Date',
+    'component-parameters': {
+      label: 'Date time picker',
+      fullWidth: true,
+      helperText: '',
+      advancedHelperText: '',
+      variant: 'outlined',
```

Comment:
I don't think this is needed and it would be good to get rid of all display config from the uiSpec.

Reply:
Follow up task as discussed.